### PR TITLE
update order driver search metadata

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/457
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/2
https://github.com/open-orchestra/open-orchestra-libs/pull/30

### CODE REVIEW

File: MongoBundle/Resources/config/metadata_search_reader.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ services:
     open_orchestra.annotation_search.driver_chain:
         class: %open_orchestra.annotation_search.driver_chain.class%
         arguments:
-            - [@open_orchestra.annotation_search.annotation_driver, @open_orchestra.annotation_search.yaml_driver, @open_orchestra.annotation_search.xml_driver]
+            - [@open_orchestra.annotation_search.yaml_driver, @open_orchestra.annotation_search.xml_driver, @open_orchestra.annotation_search.annotation_driver]
```
Comment (nicolasThal) [inline MongoBundle/Resources/config/metadata_search_reader.yml]: same for the caller

Comment (alavieille) [inline MongoBundle/Resources/config/metadata_search_reader.yml]: ??

Comment (nicolasThal) [inline MongoBundle/Resources/config/metadata_search_reader.yml]: https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/2#discussion-diff-42603719R52

**Comment by alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**